packages/docs/docs/fallback-to-canvas.mdx

```diff
@@ -7,30 +7,34 @@ Luckily this is possible using a `legacy` pixi package.
 
 ## Install
 
-Make sure to install the legacy package instead of `pixi.js`:
+If using the entire pixi package, make sure to install the legacy package instead of `pixi.js`:
 
 ```bash
 npm install pixi.js-legacy
 ```
 
-## Usage
+For advanced users using `@pixi` scoped modules, make sure to use the `canvas-` prefixed packages in place of the standard ones:
 
-Use "@pixi/react-legacy" to import pixi legacy components:
-If you want to use `"react-spring"` make sure to combine with `"@pixi/react-animated-legacy"`.
+- `@pixi/display` - `@pixi/canvas-display`
+- `@pixi/extract` - `@pixi/canvas-extract`
+- `@pixi/graphics` - `@pixi/canvas-graphics`
+- `@pixi/mesh` - `@pixi/canvas-mesh`
+- `@pixi/particle-container` - `@pixi/canvas-particle-container`
+- `@pixi/prepare` - `@pixi/canvas-prepare`
+- `@pixi/renderer` - `@pixi/canvas-renderer`
+- `@pixi/sprite-tiling` - `@pixi/canvas-sprite-tiling`
+- `@pixi/sprite` - `@pixi/canvas-sprite`
+- `@pixi/text` - `@pixi/canvas-text`
```

Comment:
```suggestion
```ts
`import "@pixi/canvas-display"`
`import "@pixi/canvas-extract"`
`import "@pixi/canvas-graphics"`
`import "@pixi/canvas-mesh"`
`import "@pixi/canvas-particle-container"`
`import "@pixi/canvas-prepare"`
`import "@pixi/canvas-renderer"`
`import "@pixi/canvas-sprite-tiling"`
`import "@pixi/canvas-sprite"`
`import "@pixi/canvas-text"`
```ts

Also note that these will not be installed for you. You will need to install them yourself
```

Not sure if the double ```ts will be an issue or not but this is a more accurate description of what they need to do